.github/workflows/publish.yml

```diff
@@ -5,22 +5,37 @@ on:
 
 jobs:
     build:
+        name: Build, Sign and Publish
         runs-on: ubuntu-latest
         permissions: write-all
```

Comment:
is it required to grant write permissions for all scopes?

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview